src/components/TodoContent.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  FormEvent, useContext, useEffect, useRef, useState,
+} from 'react';
+import { TodoList } from './TodoList';
+import { Footer } from './Footer';
+import { Todo } from '../types/Todo';
+import { ToggleAllButton } from './ToggleAllButton';
+import { ErrorContext } from '../context/ErrorContext';
+import { useLocalStorage } from '../hooks/useLocalStorage';
+
+export const TodoContent = () => {
+  const [todos, setTodos] = useLocalStorage('todos', '[]');
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>(todos);
+
+  const {
+    setIsEmptyTitleErrorShown,
+  } = useContext(ErrorContext);
```

Comment:
You can write it in one line in this case
```suggestion
  const { setIsEmptyTitleErrorShown } = useContext(ErrorContext);
```